code/mob/living/critter/small_animal.dm

```diff
@@ -72,6 +72,12 @@ ABSTRACT_TYPE(/mob/living/critter/small_animal)
 	ai_retaliate_patience = 2
 	ai_retaliate_persistence = RETALIATE_ONCE
 
+	var/aggressive = FALSE
+	var/random_name = FALSE
+	var/random_look = FALSE
+	var/name_list = "names/pets.txt"
+	var/player_can_spawn_with_pet = FALSE // For the Pet Person trait that makes you spawn with a pet (do not confuse with is_pet!!!)
```

Comment:
Also note to whoever merges this: will need secret changes to blacklist a few secret critters from appearing here.